dev/io.openliberty.grpc.1.0.internal.client/bnd.bnd

```diff
@@ -15,9 +15,10 @@
 -sub: *.bnd
 
 bVersion=1.0
-grpcVersion=1.43.2
+grpcVersion=1.57.2
```

Comment:
Update copyright date range to include 2023

Reply:
Done